gramps/gen/lib/grampstype.py

```diff
@@ -345,5 +345,11 @@ def __eq__(self, value):
     def __ne__(self, value):
         return not self.__eq__(value)
 
-    value = property(__int__, __set_int, None, "Returns or sets integer value")
-    string = property(__str__, __set_str, None, "Returns or sets string value")
+    value = property(__int__, set_int, None, "Returns or sets integer value")
+    string = property(__str__, set_str, None, "Returns or sets string value")
+
+
+@GrampsType.set.register
```

Comment:
This should be private and ideally inside the class.

Reply:
"decorators apply to the function objects before the class object is created." [Source](https://stackoverflow.com/questions/24063788/python3-singledispatch-in-class-how-to-dispatch-self-type) so I think I have to keep this as a function.
Given then that this is a function, I have to call a helper method `GrampsType.set_instance` to set the private properties of the class. If I make the method `GrampsType.set_instance` private, I can't call it from outside the class.

Reply:
This also can remain private.

Reply:
This is addressed in the most recent commit